src/createAnimatedComponent.js

```diff
@@ -250,17 +250,18 @@ export default function createAnimatedComponent(Component) {
         viewTag = findNodeHandle(this);
         viewName = null;
       } else {
+        // hostInstance can be null for a component that doesn't render anything (render function returns null). Example: svg Stop: https://github.com/react-native-svg/react-native-svg/blob/develop/src/elements/Stop.tsx
         const hostInstance = RNRenderer.findHostInstance_DEPRECATED(this);
         // we can access view tag in the same way it's accessed here https://github.com/facebook/react/blob/e3f4eb7272d4ca0ee49f27577156b57eeb07cf73/packages/react-native-renderer/src/ReactFabric.js#L146
-        viewTag = hostInstance._nativeTag;
+        viewTag = hostInstance?._nativeTag;
```

Comment:
Note: this will make `viewDescriptor.tag = undefined`. I'm not experienced enough with Rea2 to say anything so I'm just mentioning it.

Reply:
I know about that, it is handled by the checks + view descriptor AFAIK only plays a role when you use an animated style which doesn't really make sense when nothing is rendered